docroot/sites/default/services.local.yml

```diff
@@ -1,7 +1,7 @@
 # Uncomment the lines below to enable debugging.
 # Do not commit the changes on this file.
-
-#twig.config:
-#  debug: true
-#  auto_reload: true
-#  cache: false
+#parameters:
```

Comment:
This is nice! I've been looking for a spot to put this that will stick after running `ahoy comi`.